test/utils/helpers.ts

```diff
@@ -40,6 +46,40 @@ export const mockGetPaymentData = (searchQuery: string, offset: number, filterBy
 	}
 }
 
+export const filterBySchemes = (results: any, schemes: string[]) => {
+	if (!schemes || !schemes.length) {
+	  return results
+	}
+	
+	return results.filter((x: any) => schemes.includes(x.scheme))
+  }
+
+export const filterByAmounts = (results: any, amounts: string[]) => {
+	if(!amounts || !amounts.length) {
+	  return results
+	}
+  
+	const amountRanges = amounts.map(x => {
+	  const [_from, _to] = x.split('-')
+	  return { from: parseFloat(_from), to: parseFloat(_to)}
+	})
+  
+	return results.filter((x: any) => {
+	  return amountRanges.some(({ from, to }) => {
+		const totalAmount = parseFloat(x.amount)
+  
+		if(!to) {
```

Comment:
can these lines be simplified as
return (!to) ? (totalAmount >= from) : (totalAmount >= from && totalAmount <= to)